patches/server/0469-Fix-Per-World-Difficulty-Remembering-Difficulty.patch

```diff
@@ -31,6 +40,15 @@ index a36fa61313baf9ab260b8805860ac4bce24bc3c6..3ac5ab881913a90d5b59610e81d598e1
          }
      }
  
+@@ -1741,7 +1744,7 @@ public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTa
+         while (iterator.hasNext()) {
+             ServerLevel worldserver = (ServerLevel) iterator.next();
+ 
+-            worldserver.setSpawnSettings(this.isSpawningMonsters(), this.isSpawningAnimals());
++            worldserver.setSpawnSettings(worldserver.serverLevelData.getDifficulty() != Difficulty.PEACEFUL && ((DedicatedServer) this).settings.getProperties().spawnMonsters, this.isSpawningAnimals()); // Paper - per level difficulty (from setDifficulty(ServerLevel, Difficulty, boolean))
+         }
+ 
+     }
```

Comment:
The method this is in isn't used anywhere, but I guess it can't hurt to change it here as well.